apps/static/dance/ai/model/cached_palettes_map.json

```diff
@@ -1,109 +1,107 @@
-      0.757, 0.802, 0.776, 0.793, 0.778, 0.769, 0.771, 0.77, 0.785, 0.766, 0.779
+      0.253, 0.305, 0.268, 0.292, 0.339, 0.27, 0.296, 0.445, 0.298, 0.369,
+      0.074, 0.442, 0.13, 0.271, 0.301, 0.248, 0.257, 0.214, 0.267, 0.168, 0.293
     ],
     "light-bulb": [
-      0.776, 0.783, 0.799, 0.769, 0.782, 0.777, 0.795, 0.828, 0.749, 0.805,
-      0.786, 0.855, 0.777, 0.806, 0.799, 0.81, 0.799, 0.797, 0.783, 0.791, 0.789
+      0.274, 0.296, 0.174, 0.21, 0.27, 0.296, 0.333, 0.333, 0.266, 0.363, 0.248,
+      0.438, 0.2, 0.306, 0.344, 0.314, 0.245, 0.343, 0.228, 0.298, 0.357
     ],
     "robot": [
-      0.775, 0.8, 0.802, 1.0, 0.791, 0.762, 0.779, 0.83, 0.781, 0.774, 0.752,
-      0.758, 0.752, 0.767, 0.745, 0.751, 0.782, 0.769, 0.795, 0.751, 0.763
+      0.225, 0.248, 0.276, 1.0, 0.23, 0.16, 0.166, 0.391, 0.326, 0.246, 0.119,
+      0.223, 0.124, 0.176, 0.155, 0.114, 0.278, 0.203, 0.342, 0.159, 0.17
     ],
     "face-with-spiral-eyes": [
-      0.786, 0.79, 0.779, 0.78, 0.805, 0.78, 0.815, 0.783, 0.798, 0.84, 0.797,
-      0.789, 0.737, 0.785, 0.787, 0.782, 0.779, 0.777, 0.806, 0.798, 0.796
+      0.162, 0.197, 0.237, 0.214, 0.318, 0.291, 0.321, 0.228, 0.246, 0.329,
+      0.171, 0.323, 0.109, 0.238, 0.34, 0.268, 0.144, 0.218, 0.336, 0.263, 0.312
     ],
-    "pile-of-poo": [
-      0.766, 0.774, 0.809, 0.753, 0.772, 0.754, 0.767, 0.764, 0.729, 0.765,
-      0.758, 0.756, 0.741, 0.764, 0.759, 0.759, 0.776, 0.77, 0.811, 0.76, 0.749
+    "cold-face": [
+      0.329, 0.238, 0.268, 0.194, 0.177, 0.355, 0.123, 0.211, 0.319, 0.141,
+      0.321, 0.268, 0.211, 0.279, 0.303, 0.246, 0.198, 0.466, 0.212, 0.383,
+      0.743
     ],
     "artist-palette": [
-      0.769, 0.804, 0.777, 0.758, 0.811, 0.82, 0.845, 0.788, 0.76, 0.803, 0.791,
-      0.765, 0.756, 0.783, 0.757, 0.769, 0.774, 0.77, 0.775, 0.788, 0.787
+      0.275, 0.365, 0.207, 0.173, 0.496, 0.573, 0.649, 0.333, 0.194, 0.362,
+      0.268, 0.287, 0.298, 0.26, 0.329, 0.345, 0.254, 0.243, 0.23, 0.291, 0.365
     ],
     "mirror-ball": [
-      0.739, 0.785, 0.762, 0.782, 0.813, 0.771, 0.791, 0.8, 0.73, 0.794, 0.744,
-      0.761, 0.753, 0.754, 0.75, 0.76, 0.763, 0.753, 0.797, 0.758, 0.763
+      0.029, 0.188, 0.177, 0.161, 0.17, 0.254, 0.238, 0.177, 0.039, 0.282, 0.07,
+      0.11, 0.073, 0.083, 0.096, 0.139, 0.158, 0.05, 0.275, 0.094, 0.125
     ],
     "musical-note": [
-      0.752, 0.749, 0.769, 0.772, 0.752, 0.727, 0.735, 0.756, 0.727, 0.752,
-      0.747, 0.757, 0.725, 0.747, 0.724, 0.731, 0.749, 0.745, 0.754, 0.729,
-      0.725
+      0.14, 0.031, 0.127, 0.297, 0.122, 0.099, 0.088, 0.097, 0.109, 0.111, 0.08,
+      0.137, 0.038, 0.072, 0.088, 0.07, 0.099, 0.058, 0.184, 0.056, 0.068
     ],
-    "cold-face": [
-      0.79, 0.778, 0.804, 0.755, 0.775, 0.798, 0.769, 0.774, 0.784, 0.782,
-      0.788, 0.775, 0.749, 0.795, 0.776, 0.773, 0.787, 0.815, 0.772, 0.81, 0.881
+    "pile-of-poo": [
+      0.105, 0.124, 0.439, 0.142, 0.086, 0.146, 0.167, 0.139, 0.193, 0.168,
```

Comment:
`0.439`? Was this a typo? Clearly it should be `0.493`. How embarrassing!

j/k 🤣